modules/core/src/opencl/runtime/autogenerated/opencl_clamdblas_impl.hpp

```diff
@@ -338,7 +338,7 @@ namespace {
 openclamdblas_fn22(OPENCLAMDBLAS_FN_clAmdBlasCgemmEx, clAmdBlasStatus, (clAmdBlasOrder p1, clAmdBlasTranspose p2, clAmdBlasTranspose p3, size_t p4, size_t p5, size_t p6, FloatComplex p7, const cl_mem p8, size_t p9, size_t p10, const cl_mem p11, size_t p12, size_t p13, FloatComplex p14, cl_mem p15, size_t p16, size_t p17, cl_uint p18, cl_command_queue* p19, cl_uint p20, const cl_event* p21, cl_event* p22))
 clAmdBlasStatus (*clAmdBlasCgemmEx)(clAmdBlasOrder, clAmdBlasTranspose, clAmdBlasTranspose, size_t, size_t, size_t, FloatComplex, const cl_mem, size_t, size_t, const cl_mem, size_t, size_t, FloatComplex, cl_mem, size_t, size_t, cl_uint, cl_command_queue*, cl_uint, const cl_event*, cl_event*) =
         OPENCLAMDBLAS_FN_clAmdBlasCgemmEx_switch_fn;
-static const struct DynamicFnEntry clAmdBlasCgemmEx_definition = { "clAmdBlasCgemmEx", (void**)&clAmdBlasCgemmEx};
+static const struct DynamicFnEntry clAmdBlasCgemmEx_definition = { "clblasCgemmEx", (void**)&clAmdBlasCgemmEx};
```

Comment:
Patching of auto-generated files is not a good idea.
Need to patch/re-run generator with actual headers.

Reply:
Okay, I will patch parser_clamblas.py and parser_clamdfft.py and regenerate the headers.